### ISSUE TITLE
oAuth flow might be broken?

### ISSUE THREAD

**Issue by Jbarget** [CoffeeDotFilter/retweetr#39]

**Comment by oturnermajor** [comment]
Yeah this seems to happen regularly when multiple people are logging in. Any ideas why it might be? Googling gave us something about 'clock drift' but we don't know what to do about it : (

**Comment by Jbarget** [comment]
'clock drift' - never heard of it

**Comment by Jbarget** [comment]
i'll have a look but for now crack on with getting issues up on other peeps projects

**Comment by Jbarget** [comment]
This is the url where it breaks:

https://api.twitter.com/oauth/authenticate?oauth_token=

so it looks like the token isnt getting sent with the request

**Comment by franzmoro** [comment]
could you have exceded a quota?

**Comment by oturnermajor** [comment]
Should be fixed now (#40)
